validphys2/src/validphys/results.py

```diff
@@ -656,14 +677,21 @@ def one_or_more_results(
     sqrt_covmat,
     pdfs: (type(None), Sequence) = None,
     pdf: (type(None), PDF) = None,
+    unpolarized_bcs: (type(None), Sequence) = None,
```

Comment:
This is the first time that more than one unpolarized boundary condition appear, and would be inconsistent with other functions. It is preferable to remove it from existing imho

```suggestion
```

Reply:
But you drop inmediately the list and say `Boundary conditions should always be unique for a Prediction`. Therefore, don't allow more than one and you don't have to drop it.

Reply:
Maybe I am missing something, but the thing is each polarized fit card will always contain `unpolarized_bc` and during fits comparison these are passed as `unpolarized_bcs`:
https://github.com/NNPDF/nnpdf/blob/840ac3e2775929353c7e44be20fd494ff7fb1c10/validphys2/src/validphys/comparefittemplates/comparecard_polarized.yaml#L37-L39

which technically could be different but we require it to be always the same.

Reply:
Since you require them to be always the same, there should only be one. In practice here you are always taking the one from current and dropping reference, at that point is better just to do

```yaml
unpolarized_bcs:
    _from: current
```